src/theme/components/template.ts

```diff
@@ -1,113 +1,68 @@
 import { defineStyleConfig } from "@chakra-ui/react";
 import { defineStyle } from "@chakra-ui/system";
-/**
- * Grid layout based on https://www.joshwcomeau.com/css/full-bleed/
- */
+
+export const responsiveGap = { base: "1rem", md: "1.5rem", xl: "1rem" };
+export const responsivePadding = { base: "1rem", md: "1.5rem", xl: "2rem" };
```

Comment:
To keep proper alignment in the UI, this same type of responsive padding will be used in a number of other components. The responsive gaps could also be used in some places. Do you think it would make sense to move these into `global.ts`?

Reply:
Yes, could this also be a general "DS gap" and "DS padding" set of values that can be used elsewhere? Perhaps exported as design tokens if consuming apps need them.

Reply:
I was thinking about this a bit more. Design tokens are static, while this object is responsive. From what I recall, the only other design tokens that are setup with reference to screen size are the form input height values. There's one for default (anything other than mobile) and one for mobile. If we do setup tokens for these gap values, I would like to do it using the naming conventions of the new breakpoints (small mobile, large mobile, etc.).